dist/web3.js

```diff
@@ -2477,7 +2488,7 @@ var pollFilter = function(self) {
 
 };
 
-var Filter = function (options, methods, formatter) {
+var Filter = function (options, methods, callback, formatter) {
```

Comment:
It might be sensible to put `callback` at the end of params list. Also using `if (argumets...` might be good here